src/utils.js

```diff
@@ -218,7 +218,7 @@ export const paymentToString = (string) => {
         case "credit_card":
             return "Credit Card";
         default:
-            return capitalizeString(string);
+            return capitalizeString(string.toLowerCase());
```

Comment:
Did some digging and it looks like some of the queries might be using the old REST responses. Can you change the ```paymentToString``` function to take in the new graphQL responses? You can just copy and past the code below. 
```
export const paymentToString = (string) => {
    switch (string) {
        case "INTL_CREDIT_CARD":   
            return "International Credit Card"; 
        case "CREDIT_CARD":
            return "Credit Card";
        default:
            return capitalizeString(string);
    }
};
```

Reply:
I can do that, but are there pages that still get the REST responses that would break if we change this? Should I go through and check where this is used to see if changing it would cause problems?